### PR TITLE
Binding for support of MCP342X ADC's

### PR DESCRIPTION
**Pull request by falkena**
Hi @all,

I've implemented a binding for native support of MCP342X family ADC converters.
Take a look, may be someone will need them.

### CODE REVIEW

**Comment by falkena** [comment]
Documentation page will follow.

File: bundles/binding/org.openhab.binding.mcp3424/.classpath [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="src" path="src/main/java/"/>
+	<classpathentry kind="src" path="src/main/resources/"/>
+	<classpathentry exported="true" kind="lib" path="lib/pi4j-core.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/pi4j-gpio-extension.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/pi4j-service.jar"/>
```
Comment (9037568) [inline bundles/binding/org.openhab.binding.mcp3424/.classpath]: It appears that this file did not get added to the repository?

**Comment by falkena** [comment]
pi4j-service.jar was not really need. I developed binding on Raspberry with vi :-)

**Comment by falkena** [comment]
Documentation was add: https://github.com/openhab/openhab/wiki/MCP3424-Binding

**Comment by 9037568** [comment]
LGTM.

**Comment by 9037568** [comment]
Please remember to add this binding to the wiki sidebar.

**Comment by falkena** [comment]
@9037568: Thanks and added:-)

**Comment by watou** [comment]
@falkena I think I see a typo in the example [here](https://github.com/openhab/openhab/wiki/MCP3424-Binding#item-configuration).  Also, could you go into greater explanation of the meanings of the members of `address`, `pin`, `gain`, `resolution` and any other members supported?  Just to avoid future doubts or confusion?  Thanks for contributing this new binding!

**Comment by falkena** [comment]
@watou Can you explain me where the typo is? Can't see it :-(
And sure, i can add more documentation. This binding simply translates hexadecimal code states produced/taked by chip into "human" language. The main job is done by my implementation integrated into Pi4J library. May be it's very good idea, to place a link to Pi4J. May i do it?

**Comment by watou** [comment]
Sorry; I should have been more explicit and not make you hunt for it.  Apologies.

The `"` and the `}` seem to be reversed near the end of the item definitions:

> ```
> Number Test1 "Test 1" (Tests) { mcp3424="{address:6C, pin:'CH0', gain:1, resolution:12"} } 
> ```
> returns the raw conversion result on channel 0 (CH1 on datasheet) of the IC on address 0x6C 
> ```
> Dimmer Test2 "Test 2" (Tests) { mcp3424="{address:6C, pin:'CH1', gain:1, resolution:12"} } 
> ```
> returns the conversion result in percent on channel 1 (CH2 on datasheet) of the IC on address 0x6C

Also, let me suggest that set off your examples using markdown like:

\`\`\`
code example
\`\`\`

**Comment by falkena** [comment]
No problem. Good idea about markdown and thank you. I'll modify documentaion.

**Pull request merged**